package.json

```diff
@@ -6,7 +6,11 @@
   "license": "MIT",
   "homepage": "https://github.com/giakki/uncss",
   "scripts": {
-    "test": "grunt test"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "eslint": "eslint **/*.js",
```

Comment:
I'm seeing all kinds of warning with your solution while with my solution it works on *nix and Windows.